doc/segway.rst

```diff
@@ -440,6 +440,12 @@ You can supply your own custom or modified models to Segway by using the
 uses. To learn more about using GMTK to create your own models there 
 the `GMTK documentation`_.
 
+The **train** and **annotate** modes of Segway accepts models where states 
+are given labeled names, rather than using the default GMTK numeric labels. 
```

Comment:
Should briefly specify roughly "how" this is achieved with a custom model. Which key/value is necessary in the model etc.